Makefile

```diff
@@ -13,7 +13,7 @@ all: yarn gems pods bundle xcframework
 # Install JS dependencies
 #
 yarn:
-	yarn
+	yarn install
```

Comment:
This may be outside the scope of this PR, but I am skeptical about how useful it is to have a `yarn` command in the `Makefile`. In order to use this, I'd need to look up what it does whereas `yarn install` is a command that's part of JS ecosystem.

In my opinion, we shouldn't duplicate such common commands within the `Makefile`. The usefulness of `Makefile` is a lot more apparent for commands such as `bundle-android`. Even then, we are going against the JS ecosystem by having them in `Makefile` instead of `package.json` 🤷‍♂️

Reply:
> we are going against the JS ecosystem by having them in Makefile instead of package.json

IMHO this makes sense in the context of React Native – having commands that can only run _after_ `{yarn|npm} install` makes it so that you can't use `package.json` to install dependencies, so we already need something else. This isn't the case on a server, where you'd probably bundle this into a Docker container or some other kind of pre-determined runtime environment – we can assume some global dependency exists.

In the JS ecosystem, there's `yarn`, `npm`, `pnpm`, etc, and we need to be able to enforce a consistent dependency manager across the project – none of them detect the others, so they'll happily try to coexist which can result weird bugs (what happens if I run `yarn upgrade` but you want to use `pnpm` to manage dependencies? Now we're using different versions). 

Ideally I'd love a name other than `yarn`, but IMHO it should be an internal-only command – a dependency of other commands, but not designed to be called on its own.